.gitignore

```diff
@@ -2,10 +2,16 @@
 cache/
 out/
 
-# Ignores development broadcast logs
-!/broadcast
-/broadcast/*/31337/
-/broadcast/**/dry-run/
+# Ignores broadcast logs
+broadcast/
+
+# Ignores script config
+script/input/**/*.json
+!script/input/**/template-*.json
```

Comment:
Only files starting with `template-` are committed. The actual input files are derived from them.
This pattern is derived from `dss-direct-deposit`'s own [`.gitignore`](https://github.com/makerdao/dss-direct-deposit/blob/5684cd86f28f7dcd5e0528e5328fee48ba8ff964/.gitignore#L5-L7).
The deployment script uses `ScriptTools`, which requires a specific directory structure.
We've been using the same pattern is the recent repos we've been working on.
Since this is just a configuration file, it seems simpler to just copy and paste it instead of have it tailored on a per-repo basis.